packages/juce-blueprint/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./cjs/juce-blueprint.js",
   "types": "./cjs/juce-blueprint.d.ts",
   "scripts": {
-    "build": "npm run build:types && npm run build:js",
+    "build": "npm run build:js",
```

Comment:
@julianstorer, do you have an error message or anything for context here? I'd be happy to look into sorting out the root cause. Haven't hit issues with this myself and have some code that relies on the type declarations etc.

Reply:
This was Nick's suggestion. 

When I tried to install blueprint, I was just met with a wall of errors when I ran 'npm run build' for the package, and he said this was because he'd left in this temporary typescript build that he'd been experimenting with.

I don't really understand npm or what's going on here, but without the change I couldn't even get as far as building the examples.

Reply:
Alrighty then! One shall investigate.

Reply:
Ta. I suspect it's because I have very little web-dev spam installed on my machine, and something in the build is assuming that some kind of package is available which I don't have.